src/main/kotlin/net/atos/zac/zoeken/model/zoekobject/TaakZoekObject.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.zoeken.model.zoekobject
+
+import net.atos.zac.app.task.model.TaakStatus
+import nl.lifely.zac.util.NoArgConstructor
+import org.apache.solr.client.solrj.beans.Field
+import java.util.Date
+
+@NoArgConstructor // required for Java bean inspection
+data class TaakZoekObject(
+    @Field
+    var id: String,
```

Comment:
seems a bit inconsistent to use `objectId` and `id` interchangeable, or do they have different meanings?

Reply:
objectId is used in the CSV export at least.